serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/basic-integration-tests-with-restassured.adoc

```diff
@@ -1 +1,242 @@
-= Basic Integration Test with RestAssured
\ No newline at end of file
+= Basic Integration Test with REST Assured
```

Comment:
maybe add a header like this https://asciidoctor.org/docs/asciidoc-recommended-practices/#document-title ?

Reply:
Sorry @spolti. You mean the syntax, the author name, or something else?

Reply:
the author name is not needed, just the metadata and the link, similar to https://github.com/kiegroup/kogito-docs/pull/8/files#diff-db2569701a9160b1b430536d457e6df1f77ef0ad4a9f26edd8ad440ea9ed9558R1